src/gamefallout3.cpp

```diff
@@ -49,10 +49,32 @@ bool GameFallout3::init(IOrganizer* moInfo)
   return true;
 }
 
+void GameFallout3::checkVariants()
+{
+  if (QFile::exists(m_GamePath + "/Fallout3ng.exe")) {
+    setGameVariant("Low Violence");
+  } else if (QFile::exists(m_GamePath + "/Galaxy.dll")) {
+    setGameVariant("GOG");
+  } else if (QFile::exists(m_GamePath + "/FalloutLauncherEpic.exe")) {
+    setGameVariant("Epic Games");
+  } else if (m_GamePath.endsWith("Fallout 3 goty")) {
+    setGameVariant("Steam (Game of the Year)");
+  } else {
+    setGameVariant("Steam (Regular)");
+  }
+}
```

Comment:
Prefer functional style, return the variant and let the caller set it. Or change the name of the function to `checkAndSetVariants`, but returning the value is better.